src/Commands/BuildCommand.php

```diff
@@ -100,8 +100,8 @@ private function compile(string $name): BuildCommand
         }
 
         $process = new Process(
-            './box compile --working-dir="'.base_path().'" --config="'.base_path('box.json').'"',
-            dirname(dirname(__DIR__)).'/bin',
+            (! windows_os() ? './box' : 'box.bat').' compile --working-dir="'.base_path().'" --config="'.base_path('box.json').'"',
```

Comment:
just a suggestion, as i find the code quite unreadable with the ternary operator, i would prefer to have the code structured like that:

```php
<?php
if (! windows_os()) {
    $boxCommand = './box';
    $workDir = dirname(dirname(__DIR__)).'/bin'
} else {
    $boxCommand = 'box.bat';
    $workDir = base_path('vendor')
}
 
$process = new Process($boxCommand.' compile --working-dir="'.base_path().'" --config="'.base_path('box.json').'"',           $workDir, ...
```
haven't tested the this PR but `base_path('vendor')` look a little strange to me (maybe i don't understand the problem).  does this produce a correct path if called like that `php ./myproject/box/bin/console app:build`?